src/fping.c

```diff
@@ -3088,6 +3131,7 @@ void usage(int is_error)
     fprintf(out, "   -t, --timeout=MSEC individual target initial timeout (default: %.0f ms,\n", timeout / 1e6);
     fprintf(out, "                      except with -l/-c/-C, where it's the -p period up to 2000 ms)\n");
     fprintf(out, "       --check-source discard replies not from target address\n");
+    fprintf(out, "       --icmp-timestamp send ping type Timestamp Request (only if no -b specified)\n");
```

Comment:
Perhaps use the following?

`--icmp-timestamp use ICMP Timestamp instead of ICMP Echo`

I don't think that the incompatibility with setting a size needs to be mentioned in the (rather succinct) `--help` output.